README.md

```diff
@@ -43,107 +43,255 @@ When you connect your DC2 to your network, it will be using DHCP to get an IP ad
 
 2.3. Check you can connect to your DC2
 
-Open up http://DC2.local:8765 with your browser -- this should return the configuration information from your DC2.
+Open up [http://DC2.local:8765](http://DC2.local:8765) with your browser -- this should return the configuration information from your DC2.
 It can take 40 seconds for the DC2 to boot, and 30 seconds for the dc2-node script to run and Bonjour to have completed its broadcast so that loading the web page to work. You should see something like this:
 
-```
-{
-"latestVersion": "1.0.4",
-"version": "1.0.4",
-"hostname": "dc2",
-"ip": "a.b.c.d",
-"MAC": "ff:ff:ff:ff:ff:ff",
-"callHomeResponse": 200
-}
-```
+    {
```

Comment:
Should just not be indented, with the following word on the first set of backticks so that github highlights it properly:

```
```json
```